monitor/assertions.go

```diff
@@ -0,0 +1,68 @@
+// ⚡️ Fiber is an Express inspired web framework written in Go with ☕️
```

Comment:
This whole file can be removed. Use https://github.com/stretchr/testify during tests.

Reply:
[b5c4678b7d76ec59aae75fd0e492f0388e6ab5d8](https://github.com/Redish101/contrib/commit/b5c4678b7d76ec59aae75fd0e492f0388e6ab5d8)